Postman Test Suite/Record Ownership Test/Record Ownership Manual Setup Scripts.sql

```diff
@@ -0,0 +1,82 @@
+  --Client 1 should be able to full CRUD their School 1.
+  --Derived Resource -Client 1 has full CRUD - School 1
+  --Client 2 should be able to full CRUD School 2.
+  --Derived Resource -Client 2 has full CRUD - School 2
+  --Assuming Client 2 is assigned the extra ownership token, it should also be able to full CRUD School 1. 
+  --But Client 1, without the assignment to Ownership Token 2, should not be able to CRUD School 2.
+
+use EdFi_Admin_Test
+GO
+
+INSERT INTO [dbo].[OwnershipTokens] ([Description])     VALUES  ('OwnershipToken_One')
+INSERT INTO [dbo].[OwnershipTokens] ([Description])     VALUES  ('OwnershipToken_Two')
+
+DECLARE @ApiClient_ApiClientId INT;
+DECLARE @OwnershipToken_OwnershipTokenId INT;
+SELECT @ApiClient_ApiClientId = ApiClientId FROM [dbo].[ApiClients] where Name in ('255901001')
+
+SELECT @OwnershipToken_OwnershipTokenId = OwnershipTokenId FROM [dbo].[OwnershipTokens] where [Description] in ('OwnershipToken_One')
+INSERT INTO [dbo].[ApiClientOwnershipTokens]([ApiClient_ApiClientId],[OwnershipToken_OwnershipTokenId]) VALUES (@ApiClient_ApiClientId,@OwnershipToken_OwnershipTokenId)
+
+UPDATE [dbo].[ApiClients]
+   SET CreatorOwnershipTokenId_OwnershipTokenId = @OwnershipToken_OwnershipTokenId
+ WHERE ApiClientId =@ApiClient_ApiClientId
+
+
+SELECT @ApiClient_ApiClientId = ApiClientId FROM [dbo].[ApiClients] where Name in ('255901044')
+INSERT INTO [dbo].[ApiClientOwnershipTokens]([ApiClient_ApiClientId],[OwnershipToken_OwnershipTokenId]) VALUES (@ApiClient_ApiClientId,@OwnershipToken_OwnershipTokenId)
+
+SELECT @OwnershipToken_OwnershipTokenId = OwnershipTokenId FROM [dbo].[OwnershipTokens] where [Description] in ('OwnershipToken_Two')
+INSERT INTO [dbo].[ApiClientOwnershipTokens]([ApiClient_ApiClientId],[OwnershipToken_OwnershipTokenId]) VALUES (@ApiClient_ApiClientId,@OwnershipToken_OwnershipTokenId)
+
+UPDATE [dbo].[ApiClients]
+   SET CreatorOwnershipTokenId_OwnershipTokenId = @OwnershipToken_OwnershipTokenId
+ WHERE ApiClientId =@ApiClient_ApiClientId
+
+
+SELECT CreatorOwnershipTokenId_OwnershipTokenId,*  FROM [dbo].[ApiClients]  where Name in ('255901044','255901001')
+SELECT *  FROM [dbo].[ApiClientOwnershipTokens] where ApiClient_ApiClientId in ( SELECT ApiClientId  FROM [dbo].[ApiClients]  where Name in ('255901044','255901001'))
+SELECT *  FROM [dbo].[OwnershipTokens] where [Description] in ('OwnershipToken_One','OwnershipToken_Two')
+
+USE [EdFi_Security_Test]
+Go
+
+DECLARE @AuthorizationStrategyId INT;
+
+INSERT INTO [dbo].[AuthorizationStrategies]   ([DisplayName],[AuthorizationStrategyName],[Application_ApplicationId]) VALUES ('Ownership Based','OwnershipBased', 1)
```

Comment:
Shouldn't we have a place elsewhere in the change scripts for the Ownership Based authorization feature that already adds this record?

Reply:
Do you mean that under this folder where adding 'OwnershipBased' AuthorizationStrategies insert script exists here Ed-Fi-ODS\Application\EdFi.Ods.Standard\Artifacts\MsSql\Structure\Ods\Changes\

Reply:
@vimayya The security metadata for the OwnershipBased authorization strategy should be added by the feature (right?), so does that mean this `INSERT` statement should be in a script located at the following path in the Ed-Fi-ODS repository? _/Artifacts/MsSql/Data/Security/{featureName}_

Or should it be in the implementation repository (because the Ownership-based authorization feature project is in the Implementation repository)?

Reply:
Current auth strategy creation logic is in code gen https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/main/Utilities/CodeGeneration/EdFi.Ods.CodeGen/Mustache/ResourceClaimMetadata.mustache#L43 If there is no easy way include it only when the feature is on, we should probably keep it together with the rest in codegen.

Reply:
So there's no feature/extension-based sub-folder checking done for the Security database, as is done for extensions on the ODS?

Reply:
feature based sub-folder checking may work for the Security database, but we haven't done that so far.

Reply:
codegen will not  know OwnershipBasedAuthorization feature is on or not at the time of generating it .  I will have insert script in codegen